Fix comments to update mysql version in reference URL [ci skip]

giraffate opened the pull request:
### Summary
Mysql versions 5.5.8 and up are supported in rails6.

kamipo commented:
Refer latest stable, MySQL 5.5 is already EOL since December 31, 2018.

https://www.mysql.com/jp/support/eol-notice.html

giraffate committed:
Fix comments to update mysql version in reference URL [ci skip]

Mysql versions 5.5.8 and up are supported in rails6 and MySQL 5.5 is
already EOL since December 31, 2018.

giraffate commented:
> Refer latest stable, MySQL 5.5 is already EOL since December 31, 2018.

Thanks for the information.
